frontend/src/components/tutorial/Tutorial.js

```diff
@@ -138,7 +157,7 @@ export default function Tutorial({ fixedPosition, pointerRefs, nextStepTriggered
       sameSite: "lax",
     });
     const nextStep = getTutorialStepFromCookie(tutorialSteps, newCookieValue, user);
-    setStep(!isStartingStep || nextStep > 0 ? nextStep : 1);
+    handleSetStep(!isStartingStep || nextStep > 0 ? nextStep : 1);
```

Comment:
I think this could be replaced with `handleSetStep(nextStep ? nextStep : +isStartingStep);`? :-)

Reply:
(`+` is an operator that converts `boolean` to `integer`)

Reply:
This is good to know! :slightly_smiling_face:
I'm just not sure if we can use it here because `getTutorialStepFromCookie(tutorialSteps, newCookieValue, user)` can return -1 in some cases. In that case the expression `!isStartingStep || nextStep > 0 ? nextStep : 1` would become 1 and the expression you suggested -1. Since setting the step to -1 would close the tutorial it is maybe safer as it is. Or should I rather check if the return value of -1 will actually occur at this point?

Reply:
Ah, didn't know about this case. All good then :-) keep it as it is to make sure we are not introducing a new bug